src/routes/download/+page.svelte

```diff
@@ -43,35 +48,42 @@
 	<TextBlock variant="titleLarge" style="text-align: center;"
 		>{$_("download.title", defaultI18nValues)}</TextBlock
 	>
-	<InfoBar class="donation-infobar" severity="success" closable={false}>
-		<div
-			style="display: flex; gap: 0.5rem; margin-block-end: 7px; margin-block-start: 7px;"
+	{#if innerWidth > 510}
+		<InfoBar
+			style="min-height: 75px; overflow: hidden;"
+			class="donation-infobar"
+			severity="success"
+			closable={false}
 		>
-			{$_("download.donation_description", defaultI18nValues)}
-
-			<Button
-				slot="action"
-				variant="accent"
-				href="https://github.com/sponsors/yaira2"
-				{...externalLink}
+			<div
+				style="display: flex; gap: 0.5rem; margin-block-end: 7px; margin-block-start: 7px;"
 			>
-				{$_("download.donation_button", defaultI18nValues)}
-			</Button>
-		</div>
+				{$_("download.donation_description", defaultI18nValues)}
+				<Button
```

Comment:
Is there a reason why this button is no longer an accent button?

Reply:
No